src/super_gradients/training/utils/predict/predictions.py

```diff
@@ -106,3 +106,34 @@ def _validate_input(self, poses: np.ndarray, scores: np.ndarray, edge_links, edg
 
     def __len__(self):
         return len(self.poses)
+
+
+@dataclass
+class ClassificationPrediction(Prediction):
+    """Represents a Classification prediction"""
+
+    confidence: np.float32
+    labels: np.int64
+    image_shape: Tuple[int, int]
+
+    def __init__(self, confidence: np.float32, labels: np.int64, image_shape: Optional[Tuple[int, int]]):
+        """
+
+        :param confidence:  Confidence scores for each bounding box
+        :param labels:      Labels for each bounding box.
+        :param image_shape: Shape of the image the prediction is made on, (H, W).
+        """
+        self._validate_input(confidence, labels)
+
+        self.confidence = confidence
+        self.labels = labels
+        self.image_shape = image_shape
```

Comment:
FYI `@dataclass` actually provide a `__post_init__` method meant for this type of thing. The advantage is that it doesnt override the default `@dataclass` `__init__`, so you don't need to write all this `self.confidence = confidence` logic.

I think this is the syntax
```python
def __post_init__(self, confidence: np.float32, labels: np.int64, image_shape: Optional[Tuple[int, int]]):
    self._validate_input(confidence, labels)
```